docs/introduction.asciidoc

```diff
@@ -5,29 +5,31 @@ https://www.elastic.co/guide/en/apm/agent/ruby/current/introduction.html[elastic
 endif::[]
 
 [[introduction]]
-
 == Introduction
 
-Welcome to the APM Ruby Agent documentation.
-
-The Elastic APM Ruby Agent sends performance metrics and error logs to an
-Elastic APM Server.
+The Elastic APM Ruby Agent sends performance metrics and error logs to the APM Server.
 It has built-in support for <<getting-started-rails,Ruby on Rails>> and other
 <<getting-started-rack,Rack-compatible>> applications.
+It also offers an API which allows you to instrument any application.
+
+[float]
+[[how-it-works]]
+=== How does the Agent work?
+
+The agent auto-instruments <<supported-technologies,supported technologies>> and records interesting events,
+like HTTP requests and database queries. To do this, it {NEED CONTENT HERE}.
+This means that for the supported technologies, there are no code changes required.
+
+The Agent automatically {NEED CONTENT HERE} to measure their duration and metadata (like the DB statement),
```

Comment:
```suggestion
The Agent automatically keeps track of your queries to your data stores to measure their duration and metadata (like the DB statement),
```

Reply:
I suggest `keeps track of queries to your data stores`

Reply:
Same as above. GH is fancy!

Reply:
I removed the first `your` ; )